spec/term/1.0.md

```diff
@@ -0,0 +1,116 @@
+For example, the position `(0,0)` refers to the character at the top left corner of the array.
+
+For graphical terminals, the character array MAY be much larger than what can be displayed on screen at once.
+In this case, the character array itself is called the terminal's **scrollback**, and the range of currently visible columns is called the terminal's **viewport**.
+The number of columns in the viewport is called the terminal's **viewport height**.
+
+*Rationale:* The term "terminal height" is not defined to avoid confusion. We always refer explicitly to either the scrollback height or the viewport height.
+
+### 1.2. Standard output
+
+When a client writes a byte string into its standard output, the server SHALL assume that this byte string is encoded in [UTF-8](https://tools.ietf.org/html/rfc3629), and decode it into a string of Unicode characters.
+
+When the `term1.output-protected` property is true, the server MUST discard all characters from this string that are in the general categories Cc ("Other, control"), Cf ("Other, format"), Cs ("Other, surrogate"), Co ("Other, private use") and Cn ("Other, not assigned"), except for U+0009 (Horizontal Tab), U+000A (New Line) and U+000D (Carriage Return).
+
+When the `term1.output-protected` property is false, the server MAY  alternatively:
+
+- choose a printable representation for each such character and replace the character with that representation before continuing, or
+- interpret the character as starting a control sequence if the character is U+001B (Escape).
+
+When a control character is interpreted as starting a control sequence, the entire control sequence SHALL be removed from the string before processing as described in this section continues, and the remaining string MAY be modified depending on the semantics of the control sequence.
+
+*Rationale:* This rule allows terminals to be backwards-compatible with legacy control sequences, such as those defined in VT100 aka ECMA-48 aka ANSI X3.64 aka ISO/IEC 6429.
+
+Characters that are in the general category Co ("Other, private use") MAY be exempted from the treatment described above if the characters in question have a graphic representation in the font that the terminal uses to display its character array.
+
+Finally, the server MUST discard each U+000A (new line) character that immediately succeeds a U+000D (carriage return) character, and then convert each U+000D character into a U+000A character.
+
+*Rationale:* This rule ensures consistent line break handling across different platforms, i.e., CR+NL = CR = NL.
+
+The remaining string of Unicode characters SHALL then be written into the character array starting from the position pointed to by the cursor, using the rules for normalization, writing direction, line breaking and handling of bidirectional text as laid out in the Unicode Standard, as published by the [Unicode consortium](https://www.unicode.org).
```

Comment:
Note to self: Should check how the Unicode standard is referenced in other standards that concern text layouting, e.g. HTML/CSS.